src/api.js

```diff
@@ -4,10 +4,16 @@ const newsApi = axios.create({
   baseURL: "https://jacobs-news-api.onrender.com/api",
 });
 
-export const getArticles = () => {
-  return newsApi.get("/articles").then((res) => {
-    return res.data;
-  });
+export const getArticles = (query) => {
+  if (query) {
+    return newsApi.get(`/articles?topic=${query}`).then((res) => {
```

Comment:
Axios can handle this for you https://notes.northcoders.com/courses/js-front-end/axios#axios-params